airbyte-ci/connectors/connector_ops/connector_ops/utils.py

```diff
@@ -229,6 +229,9 @@ def get_all_gradle_dependencies(
         if dependency_path not in found_dependencies and Path(dependency_path / "build.gradle").exists():
```

Comment:
maybe something like this? to avoid the mostly-identical branches
```python
valid_build_files = ["build.gradle", "build.gradle.kts"]
for build_file in valid_build_files:
  if dependency_path not in found_dependencies and Path(dependency_path / build_file).exists():
    ...
```

Reply:
Done